masterlist.yaml

```diff
@@ -25823,73 +25858,82 @@ plugins:
       - Graphics
       - Names
       - Relev
-    dirty:
-      - <<: *quickClean
-        crc: 0x1ABE5A11
-        util: '[TES4Edit v4.0.2i](https://www.nexusmods.com/oblivion/mods/11536)'
-        itm: 3
+    clean:
+      - crc: 0x10792EEA
+        util: 'TES4Edit v4.0.4c'
   - name: 'WAC - Legion.esp'
     tag:
       - Graphics
-      - Invent
+      - Invent.Add
+      - Invent.Remove
     dirty:
       - <<: *quickClean
         crc: 0x062F45EE
-        util: '[TES4Edit v4.0.2i](https://www.nexusmods.com/oblivion/mods/11536)'
+        util: '[TES4Edit v4.0.4c](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 3
   - name: 'WAC - Magic.esp'
-    dirty:
-      - <<: *quickClean
-        crc: 0x290989F0
-        util: '[TES4Edit v4.0.2i](https://www.nexusmods.com/oblivion/mods/11536)'
-        itm: 3
+    tag: [ SpellStats ]
+    clean:
+      - crc: 0xD6F4D75A
+        util: 'TES4Edit v4.0.4c'
   - name: 'WAC - OverSpawn.esp'
     tag:
       - Actors.ACBS
+      - Actors.Anims
+      - Actors.Skeleton
       - Actors.Spells
       - Actors.Stats
       - Delev
+      - EffectStats
       - Graphics
-      - Invent
-      - Relations
+      - Invent.Add
+      - Invent.Remove
+      - NPC.FaceGen
+      - NPC.Hair
       - Relev
     dirty:
       - <<: *quickClean
-        crc: 0x030C2196
-        util: '[TES4Edit v4.0.2i](https://www.nexusmods.com/oblivion/mods/11536)'
-        itm: 6
+        crc: 0x9C82C093
+        util: '[TES4Edit v4.0.4c](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 3
   - name: 'WAC - Player Summons.esp'
     dirty:
       - <<: *quickClean
-        crc: 0xEA321AD6
-        util: '[TES4Edit v4.0.2i](https://www.nexusmods.com/oblivion/mods/11536)'
-        itm: 30
+        crc: 0x77FF1D67
+        util: '[TES4Edit v4.0.4c](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 28
   - name: 'WAC - TCOS.esp'
     tag:
       - Actors.ACBS
+      - Actors.Spells
       - Actors.Stats
       - Delev
-      - Eyes
       - Graphics
-      - Invent
+      - Invent.Add
+      - Invent.Remove
+      - Names
+      - NPC.FaceGen
+      - NPC.Hair
       - NPC.Race
-      - R.Teeth
       - Relev
-    dirty:
-      - <<: *quickClean
-        crc: 0xFDBEF368
-        util: '[TES4Edit v4.0.2i](https://www.nexusmods.com/oblivion/mods/11536)'
-        itm: 3
+      - R.Eyes
+      - R.Teeth
+      - Stats
+    clean:
+      - crc: 0x39FC05A7
+        util: 'TES4Edit v4.0.4c'
   - name: 'WAC - TCOS Shops.esp'
     tag:
-      - Graphics
-      - Invent
-    dirty:
-      - <<: *quickClean
-        crc: 0xBBC15780
-        util: '[TES4Edit v4.0.2i](https://www.nexusmods.com/oblivion/mods/11536)'
-        itm: 3
-  ### WAC - Item & Loot Fix ###
```

Comment:
I'd either keep the subsection headers or sort the rest of these optional plugins alphabetically with the rest.